### ISSUE TITLE
IOError when trying to install something on 64bit Ubuntu

### ISSUE THREAD

**Issue by Brainzyy** [PlayOnLinux/POL-POM-5#427]
It says it can't find 'wine' which is very strange.
Running as root doesn't help either.
Backtrace:

```
[JavaFX Application Thread] ERROR com.playonlinux.javafx.views.common.ErrorMessage - java.lang.RuntimeException: java.io.IOException: Cannot run program "/home/gijs/.PlayOnLinux-5/engines//wine//upstream-linux-x86/1.8.6/bin/wine" (in directory "/home/gijs/.PlayOnLinux-5/wineprefix/Notepad++/drive_c"): error=2, No such file or directory
	at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:397)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:446)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:403)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:399)
	at jdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:155)
	at javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
	at com.playonlinux.scripts.nashorn.NashornEngine.eval(NashornEngine.java:27)
	at com.playonlinux.scripts.nashorn.NashornScriptInterpreter.runScript(NashornScriptInterpreter.java:17)
	at com.playonlinux.scripts.interpreter.BackgroundScriptInterpreter.lambda$runScript$0(BackgroundScriptInterpreter.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Cannot run program "/home/gijs/.PlayOnLinux-5/engines//wine//upstream-linux-x86/1.8.6/bin/wine" (in directory "/home/gijs/.PlayOnLinux-5/wineprefix/Notepad++/drive_c"): error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at jdk.nashorn.internal.scripts.Script$Recompilation$73$4016AAA$\^eval\_.run(<eval>:202)
	at jdk.nashorn.internal.scripts.Script$Recompilation$72$10114A$\^eval\_.regedit$open(<eval>:415)
	at jdk.nashorn.internal.scripts.Script$Recompilation$62$167$\^eval\_.luna(<eval>:23)
	at jdk.nashorn.internal.scripts.Script$Recompilation$38$704$\^eval\_.go(<eval>:41)
	at jdk.nashorn.internal.scripts.Script$18$\^eval\_.:program(<eval>:3)
	at jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:637)
	at jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:494)
	at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:393)
	... 11 more
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 19 more
```

**Comment by plata** [comment]
This _could_ also be a problem with `/home/gijs/.PlayOnLinux-5/wineprefix/Notepad++/drive_c` instead of wine.

**Comment by Brainzyy** [comment]
@plata @qparis I tried changing the install directory (#436) but it gave me the same error.

**Comment by Brainzyy** [comment]
@qparis @plata I have good news and I have bad news. I've managed to fix this issue. But the weird thing is, I had to install wine-devel and all of it's dependencies for it to work.
There is another thing that is weird. I installed 64bit Ubuntu 16.04 onto my main rig today, and POL didn't work either. I didn't get the IO error, but there was some error about missing FreeType libs and ncurses libs. This caused a problem because none of the installers would show up. The icons were in the dock, but their windows wouldn't appear. I installed wine-devel and it's dependencies on that rig as well and then it worked fine.
The instructions for wine also said to enable i386 packages.
TLDR: It seems that POL-POM-5 needs wine-devel dependencies and/or wine-devel itself to be installed on 64bit Ubuntu machines, because on my 32bit Ubuntu 16.04 machine I didn't need any of this, POL-POM-5 worked out of the box.

**Comment by qparis** [comment]
Yes that makes sense. You need to have 32bits dependencies libraries installed so that wine runs.

**Comment by plata** [comment]
Should this be added to [wiki: Building](https://github.com/PlayOnLinux/POL-POM-5/wiki/Building)?

**Comment by Brainzyy** [comment]
@plata Yeah, probably. Otherwise people with amd64 systems will be clueless (like me).

Steps I took to fix it:

1. sudo dpkg --add-architecture i386 
2. sudo add-apt-repository ppa:wine/wine-builds
3. sudo apt-get update
4. sudo apt-get install --install-recommends winehq-devel

**Comment by plata** [comment]
@Brainzyy do you really need the PPA or is it enough to run:

```bash
sudo dpkg --add-architecture i386
sudo apt update
sudo apt install wine
```

**Comment by qparis** [comment]
You do not need the whole wine devel stuff. All 32bitz dependencies are sufficient enough

**Comment by Brainzyy** [comment]
@plata I don't think that is enough, but I will test it.
@qparis The problem is there's no good way to only install the dependencies of a package.

**Comment by qparis** [comment]
Dependencies of 32bits package of wine should be sufficient

**Comment by Brainzyy** [comment]
@qparis Yes I know, but installed only dependencies is not possible (efficiently)

**Comment by Brainzyy** [comment]
@qparis @plata Only running the commands you suggested does not work. And like I said before, only installing the dependencies is not possible. So the full wine-devel package has to be installed. (at least until we figure out exactly which dependencies are needed)

**Comment by qparis** [comment]
I'm talking about 32bit ones

**Comment by Brainzyy** [comment]
@qparis Yes I know, but how would you install only the necessary dependencies without installing the whole package (which is 32bit anyway)

**Comment by qparis** [comment]
apt-get install wine:i386 ?

**Comment by Brainzyy** [comment]
@qparis It doesn't matter because if you install wine-devel after you've enabled i386 it only installs 32bit wine. It only installs 32bit dependencies as well (because almost all 64bit dependencies come pre-installed on a 64bit system)

**Comment by qparis** [comment]
No it also install development dependencies (a.k build-essential, *-dev, ...) which are unnecessary for the end user

**Comment by Brainzyy** [comment]
@qparis Okay, my bad. I'll test your command and see if it is enough.
